src/coverlet.core/Helpers/InstrumentationHelper.cs

```diff
@@ -3,12 +3,14 @@
 using System.IO;
 using System.Linq;
 using System.Reflection.PortableExecutable;
+using Microsoft.Extensions.FileSystemGlobbing;
 
 using Coverlet.Core.Instrumentation;
+using Microsoft.Extensions.FileSystemGlobbing.Abstractions;
 
 namespace Coverlet.Core.Helpers
 {
-    internal static class InstrumentationHelper
+    public static class InstrumentationHelper
```

Comment:
@tonerdo maybe`[InternalsVisibleToAttribute]`?

Reply:
Please lets leave this as `internal`